docs/advanced/naming.md

```diff
@@ -1,190 +1,262 @@
+  - Use `.` or `-` for delimiters
 
-Examples:
+## File Names
 
-```
-[Group] Anime Name - S01E01 - (BD 1080p HEVC FLAC) [Dual Audio] [CRC32].mkv
-Anime Name - S01E01 - (BD 1080p HEVC FLAC) [Dual Audio] [CRC32]-Group.mkv
-Anime Name - 18.5 - S00E05 - (BD 1080p HEVC FLAC) [Dual Audio]-Group.mkv
-Anime Name (2022) - S01E01 - (BD Remux 1080p HEVC FLAC) [Dual Audio]-Group.mkv
-Anime.Name.S01E01.1080p.BluRay.Opus2.0.x264-Hi10P-Group.mkv
-```
+In general, a good file name should:
 
-### Specials/OVA Filename
+- Follow the same [general guidelines](#general) for naming
+- Include the **season/episode number** (for TV series)
+  - Use `SXXEXX` format (e.g. Season 2 Episode 10 = `S02E10`)
+  - Absolute episode numbers may be used together (e.g. `S02E10 - E22`), *but never by themselves*
+  - For specials, use season 0 (`S00EXX`) for guaranteed automatic parsing and correct identification. *However, it is suggested to add an additional identifier such as including `OVA` or the episode title (i.e. `S00E01 - OVA Title`) to easily differentiate between episodes*
+- Include a **releaser/group tag**
+  - This should be placed at the start or end of the file name
 
-- These are wildly inconsistent across different databases which means you need to be more careful when naming these to avoid any confusion while also keeping it both human and machine readable. Databases like [thetvdb](https://thetvdb.com/) use `S00EXX` for specials while databases like [MAL](https://myanimelist.net/) simply use the episode name for specials.
-- A common thing in anime releases is Specials/OVAs numbered as `0.5`, e.g. `Anime Name - 18.5`. This is not recommended at all.
-- `0.5` Episodes are not supported by a single piece of software or database but they usually make it easier for the person watching to know where it chronologically lands in the show. You may or may not include this in the filename.
-- `S00EXX` isn't very user friendly, because it doesn't really tell much on its own but it guarantees automatic parsing and correct identification. You must include this in the filename. Refer to [thetvdb](https://thetvdb.com/) to find this for your episode.
-- Including the Episode title is highly recommended in this case as it makes identification significantly easier and understandable for anyone who downloads it. Neither `S00EXX` or `0.5` provides this information at a glance which is why the episode title is really helpful in this case. Refer to either [thetvdb](https://thetvdb.com/) or [MAL](https://myanimelist.net/) to find the episode name.
-- You may include the absolute number of the episode as long as it's not a decimal episode that lands between any two existing ones. Optional and really just down to preference. It's much better than decimal episode numbering but still not ideal.
-- This means we need a name that is easily readable, parsable, and identifiable. This is achieved by using a combo of the above.
```

Comment:
Specials are hard to get right, which is why this section was big. I think the new one dumbed it down too much and it should be expanded a bit more to cover the deleted part